contrib/kube-prometheus/manifests/0prometheus-operator-0alertmanagerCustomResourceDefinition.yaml

```diff
@@ -11,7 +11,7 @@ spec:
   scope: Namespaced
   validation:
     openAPIV3Schema:
-      description: Alertmanager describes an Alertmanager cluster.
+      # description: Alertmanager describes an Alertmanager cluster.
```

Comment:
I commented it out in case it needs to be added again.

Reply:
Can remove it as well, let me know what you prefer. :)

Reply:
@nikhita All of these are auto-generated. The source is [here](https://github.com/nikhita/prometheus-operator/blob/1c2f14c2b216fd0ecfa9e9823466f2e133ce0aa1/pkg/client/monitoring/v1/types.go#L457).

I guess removing the comment and running `make generate` would do the job.

Thanks a lot for the help!